gtsfm/loader/astronet_loader.py

```diff
@@ -253,3 +262,21 @@ def is_valid_pair(self, idx1: int, idx2: int) -> bool:
             validation result.
         """
         return super().is_valid_pair(idx1, idx2) and abs(idx1 - idx2) <= self._max_frame_lookahead
+
+
+def get_nonzero_intensity_mask(img: Image, eps: int = 5, kernel: Tuple[int, int] = (15, 15)) -> np.ndarray:
+    """Generate mask of where image intensity values are non-zero.
```

Comment:
Let's mention that we are using a morphological operator on top of the thresholding in the docstring, e.g.

```python
"""
...
After thresholding, we use an erosion kernel to .... because ...
```

Without the erosion kernel, is the result worse? or maybe we can add a brief justification here in the docstring

Reply:
I erode the mask to add a small buffer, because theoretically there could still be border points if I didn't.

Reply:
Added explanation